examples_extra/ASDKgram-Swift/ASDKgram-Swift/OrderedDictionary/OrderedDictionary.swift

```diff
@@ -0,0 +1,610 @@
+/// A generic collection for storing key-value pairs in an ordered manner.
```

Comment:
I think we need to include copyright and license notices of the [original author](https://github.com/lukaskubanek/OrderedDictionary/) here.

Reply:
Yep I will change that, but also ask him if he can merge in the Cocoapods PR they have open so we can just pull it in via that.